### PR TITLE
reordering GCP configuration steps

### LINKED ISSUE #17390: [enterprise-4.2] Edit suggested in file installing/installing_gcp/installing-gcp-account.adoc
### Which section(s) is the issue in?

left hand nav bar

### What needs fixing?

left hand nav bar calls this page "installing GCP account", but it should be "installing GCP project"

### LINKED ISSUE #17391: [enterprise-4.2] Edit suggested in file installing/installing_gcp/installing-gcp-account.adoc
### Which section(s) is the issue in?

Enabling API services

### What needs fixing?

This section should come before the DNS section, as you need to enable the DNS API. This section can go first.

### PR DESCRIPTION
**Pull request by kalexand-rh**
Fixes #17391
Fixes #17390

@wmengRH, will you PTAL?

### CODE REVIEW

**Comment by wmengRH** [comment]
LGTM.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL?

File: modules/installation-gcp-project.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-project_{context}"]
+= Creating a GCP project
+
+To install {product-title}, you must create a project your Google Cloud Platform (GCP) account to host the cluster.
```
Comment (bergerhoffer) [inline modules/installation-gcp-project.adoc]: Something off about the sentence... "create a project for your..."?

**Review by bergerhoffer — COMMENTED** [review]
Just one thing, otherwise LGTM!

**Pull request merged**